src/pages/token/rewards/StakingPanel.tsx

```diff
@@ -223,7 +230,15 @@ const $Label = styled.div`
   gap: 0.5rem;
 `;
 
-const $BalanceOutput = styled(Output)`
+const $BalanceOutput = styled(Output)<{ isPositive: boolean }>`
```

Comment:
can we infer the color based on the raw value? 
like 
${({ value})}=> value.gt(0) ... or something?

Reply:
ehh hm as personal pref I like to keep the logic in the typescript portion (as opposed to css) but lmk if u feel strongly 👀